src/test/java/StringManipulationTest.java

```diff
@@ -1,20 +1,18 @@
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
 
+import org.junit.*;
```

Comment:
"package org.junit does not exist" this might be causing the error for "cannot find symbol"

Reply:
I added this to your StringManipulationTest. java to see if I could fix the problem and it seem too. 

import static org.junit.jupiter.api.Assertions.*;

Causes the test cases to pass.

Reply:
![Screenshot 2023-06-17 at 6 39 17 PM](https://github.com/SWE-CS410/5541-Spring23CodeReviewActivity/assets/132320379/75311e19-419a-4303-8ea6-17e4aa5a8ef1)

here's a screenshot on what I got through  the verify with Maven

Reply:
Thanks for reviewing. I'll make the proposed change.